site/zenodo_rdm/config.py

```diff
@@ -163,228 +176,20 @@
 #: Email title template.
 SUPPORT_EMAIL_CONFIRM_TITLE_TEMPLATE = "zenodo_rdm/email_confirm_title.html"
 
+# Search query of recent uploads
+# Defaults to newest records search
+ZENODO_FRONTPAGE_RECENT_UPLOADS_QUERY = "/api/records?sort=newest&size=10"
```

Comment:
There is no `ext.py` in the `zenodo_rdm` module. Therefore, there is no need to prefix this variables. If we go for prefixing for distintcion, the `SUPPORT` ones should be `ZENODO_SUPPORT_...`